sdk/typespec/typespec_client_core/src/http/response.rs

```diff
@@ -214,9 +214,8 @@ impl<T> fmt::Debug for Response<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("Response")
             .field("status", &self.status)
-            .field("headers", &self.headers)
-            .field("body", &"(body)")
```

Comment:
You could keep the `body` field, right? It's already not actually displaying the body.

Reply:
Yes, but I felt "(body)" isn't any more useful so the `finish_non_exhaustive()` that "prints" the string ".." seemed appropriate.

Reply:
I'll probably do the same with `Display` as well, since we need to implement `Display` to satisfy the `Error` trait constraint.